### PR TITLE
Added -> Did you mean?

### PR DESCRIPTION
**Pull request by TheMrPhantom**

### CODE REVIEW

File: drink.py [hunk 1]
```diff
@@ -21,14 +21,17 @@
...
 import sys
+import levenshtein as LD
 
 cfg = None
 
 def get_login_token(password):
     global cfg
-    response = requests.post(cfg['url'] + "/login", data = {'password' : password})
+    response = requests.post(
+        cfg['url'] + "/login", data={'password': password})
```
Comment (Vogel612) [inline drink.py]: why the whitespace change?

File: drink.py [hunk 2]
```diff
@@ -21,14 +21,17 @@
...
     if not response.ok:
-        print("Failed to get token: "+str(response.status_code)+": "+r.text, file=sys.stderr)
+        print("Failed to get token: "+str(response.status_code) +
+              ": "+r.text, file=sys.stderr)
```
Comment (Vogel612) [inline drink.py]: operator spacing is inconsistent, maybe even use a format-string instead?

File: drink.py [hunk 3]
```diff
@@ -39,16 +42,19 @@ def refresh_token():
...
-    r = requests.get(cfg["url"] + suburl, headers={'X-Auth-Token' : cfg['token']})
+    r = requests.get(cfg["url"] + suburl,
+                     headers={'X-Auth-Token': cfg['token']})
     if r.status_code == 403 and retry:
         refresh_token()
         return get(suburl, False)
     if not r.ok:
-        print("API returned error "+str(r.status_code)+": "+r.text, file=sys.stderr)
+        print("API returned error "+str(r.status_code) +
+              ": "+r.text, file=sys.stderr)
```
Comment (Vogel612) [inline drink.py]: again, operator spacing is inconsistent

File: drink.py [hunk 4]
```diff
@@ -60,15 +66,42 @@ def order_drink(drink, retry=True):
     if r.status_code == 403 and retry:
         refresh_token()
         return order_drink(drink, retry=False)
+
     if not r.ok:
-        print(str(r.status_code) + ": " + r.text, file=sys.stderr)
-        sys.exit(1)
-    print(r.text)
+        if r.status_code == 400:
```
Comment (Vogel612) [inline drink.py]: 400 is a magic number

File: drink.py [hunk 5]
```diff
@@ -60,15 +66,42 @@ def order_drink(drink, retry=True):
...
+
+            print("Did you mean ", end="")
+            print(correctName, end="")
+            print("? (y/n)")
+
+            answer = input()
+
+            if answer == "y":
+                order_drink(correctName)
+        else:
```
Comment (Vogel612) [inline drink.py]: consider inverting the corresponding if and returning to save one level of indentation

File: drink.py (hunk 4 above)
Comment (TheMrPhantom) [reply drink.py]: fixed: now checking for "Unknown beverage" instead of 400

File: drink.py (hunk 5 above)
Comment (TheMrPhantom) [reply drink.py]: done

File: drink.py (hunk 1 above)
Comment (TheMrPhantom) [reply drink.py]: auto-format, now fixed

File: drink.py (hunk 3 above)
Comment (TheMrPhantom) [reply drink.py]: fixed

**Comment by TheMrPhantom** [comment]
Does this pull request fix #7 ?

**Comment by Vogel612** [comment]
@TheMrPhantom as it currently stands it doesn't. Assuming the merge-commit is changed to include `fix(es)? #7` or `closes? #7` or another "triggering phrase" it will.

**Pull request merged**